Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:latest
+
+WORKDIR /
+
+COPY . .
+
+RUN npm install 
```

Comment:
if makefile runs npm install then there is no need for this line

Reply:
The full version of `node` is quite heavy, if we can get away with the alpine version, it would mean a much smaller docker image size. The only caveat is we _may_ need to install git on the alpine version, as it does't come with that.